src/test/util/setup_common.cpp

```diff
@@ -79,6 +79,18 @@ const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
 /** Random context to get unique temp data dirs. Separate from g_insecure_rand_ctx, which can be seeded from a const env var */
 static FastRandomContext g_insecure_rand_ctx_temp_path;
 
+std::ostream& operator<<(std::ostream& os, const arith_uint256& num)
+{
+    os << ArithToUint256(num).ToString();
```

Comment:
Is the `ArithToUint256` conversion necessary?

Reply:
Good point, I missed the `base_uint::ToString()` method. Not as familiar with `arith_uint256` as I am with `uint256` yet.

Should probably also document endianness of `arith_uint256` strings to mirror the added comments to `uint256` in this PR.

Reply:
Follow-up fixing this issue: #30526